mora/service/common.py

```diff
@@ -523,6 +523,53 @@ def create_organisationsenhed_payload(
     return org_unit
 
 
+def create_bruger_payload(
+    valid_from: str,
+    valid_to: str,
+    brugernavn: str,
+    brugervendtnoegle: str,
+    tilhoerer: str,
+    cpr: str
+):
+
+    virkning = _create_virkning(valid_from, valid_to)
+
+    user = {
+        'note': 'Oprettet i MO',
```

Comment:
`Importeret fra ServicePlatformen`?

Reply:
As this code is triggered by an action in MO, and not an import from SP, I feel this is fine as it is.